### ISSUE TITLE
Libnd4j: Batch norm backprop not implemented/enabled

### ISSUE THREAD

**Issue by AlexDBlack** [deeplearning4j/deeplearning4j#5439]
https://github.com/deeplearning4j/deeplearning4j/blob/74bd70162979bace33cf28b160aeb61131e13bcc/libnd4j/include/ops/declarable/generic/nn/batchnorm.cpp#L48

**Comment by shyrma** [comment]
Alex, this bp op is already present and tested on master

**Comment by AlexDBlack** [comment]
Thanks, we can close this.
I forgot about this issue. I think we even already have some tests using it...
